packages/transport/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@trezor/transport",
+  "version": "1.0.0",
+  "description": "Low level library facilitating protocol buffers based communication with Trezor devices",
+  "main": "./lib/index.js",
+  "files": [
+    "lib/"
+  ],
+  "scripts": {
+    "lint": "eslint src",
+    "type-check": "tsc --project tsconfig.json",
+    "build:lib": "rm -rf lib && tsc --p ./tsconfig.lib.json",
+    "publish:lib": "./scripts/publish-lib.sh",
+    "test:unit": "jest",
+    "example:bridge": "jest --verbose -c jest.config.e2e.js --testPathPattern bridge.integration"
+  },
+  "repository": "https://github.com/trezor/trezor-link.git",
```

Comment:
update the references from trezor-link to @trezor/transport please

Reply:
ah, good point. and also remove `example:bridge` command for now. I might add this in another branch.